src/Student1/Ex2/Nav.scss

```diff
@@ -0,0 +1,10 @@
+* {
```

Comment:
This "select all" selector is duped in `Post.scsss`, so we should remove one.

Reply:
Made the Changes